### PR TITLE
Clear all

### PR DESCRIPTION
**Pull request by DilsherB**
Clear all functionality added.

### CODE REVIEW

File: src/clearAll.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import toDoTasks from './index.js'; // eslint-disable-line
+
+const clearAll = document.querySelector('.clearAll');
+clearAll.addEventListener('click', () => {
+  const filteredData = toDoTasks.filter((e) => e.completed === false);
+  localStorage.setItem('newTodo', JSON.stringify(filteredData));
+  window.location.reload();
+});
```
Comment (franclobo) [inline src/clearAll.js]: - [ ] I have conned the repo and open in the `clearAll` branch then I run `npm i` and `npm start` but your app is not working, the add, edit, delete and delete all functions are broken, and also the local storage is not saving the information. Please 🙏🏽 fix them all for a better user experience. 👇🏽 

https://user-images.githubusercontent.com/58642949/215275840-48fb9105-848f-4d94-b970-8141373442b5.mp4

File: src/crud.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import toDoTasks from './index.js'; // eslint-disable-line
+
+document.querySelector('.fa-angle-down').addEventListener('click', () => {
+  const todoData = document.querySelector('.newTodo').value;
+  if (todoData.trim() === '') {
+    return;
+  }
+  toDoTasks.push({ description: `${todoData}`, completed: false, index: toDoTasks.length });
+  localStorage.setItem('newTodo', JSON.stringify(toDoTasks));
+  window.location.reload();
```
Comment (franclobo) [inline src/crud.js:10]: - [ ] The local storage is not saving the info, please 🙏🏽 fix it for a better user interface.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @DilsherB, 

# ♻️ Required Changes ♻️

soClose

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Github flow ✔️ 
- Professional README file ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please 🙏🏽 review that all the functionalities of your app are working, I tried it with out success.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/clearAll.js (hunk 1 above)
Comment (DilsherB) [reply src/clearAll.js]: @franclobo. Thanks for your kind review.
before submitting it i checked and it was working but what happened now I am unable to understand. can you help please.

File: src/crud.js (hunk 2 above)
Comment (DilsherB) [reply src/crud.js:10]: @franclobo. thanks for your kind review. before submitting it was working well but what happened after that i am unable to understand. can you help me.

File: src/clearAll.js (hunk 1 above)
Comment (DilsherB) [reply src/clearAll.js]: https://www.loom.com/share/5752fa58c87f49de9c221a49beb63f41

please check mr @franclobo

**Review by huseyinbiyik — APPROVED** [review]
Hi @DilsherB,

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text



## Highlights
- Linter set up well and checks pass 🟢
- Add and remove functions work fine 👌

## Optional suggestions
- Please make sure you can add new task to your to do by pressing 'ENTER' key. Each time I needed to click on the icon I marked below and this might create a bad UX. 🦾
Screenshot 2023-01-28 at 11 49 13 PM

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**